### DISCUSSION TITLE
nuxt2

### DISCUSSION THREAD

**Discussion by zxx00** [nuxt/nuxt#24386] [Questions (Nuxt 2)]
有爬虫记录，就是不被收录

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction) or more info? 🙏

**Reply by zxx00** [reply]
Hey! We have conducted many experiments, but they cannot be included in Baidu Browser. Can I provide you with a code?

**Reply by zxx00** [reply]
[Uploading benhu-official-web-dev-1.0.1.zip…]()
